.gitignore

```diff
@@ -59,3 +59,9 @@ typings/
 
 # next.js build output
 .next
+
+# software development files
+.idea
+
+# macos files
+.DS_Store
```

Comment:
Ok, 
But files related to environment are generally not added to `.gitignore` (but to a global one set in the machine), else we should add those of other operating systems

Reply:
Ça me va de l'enlever si on est sûr de pas se retrouver avec des fichiers moisis a l'intérieur, je trouve ça un peu plus simple plutôt que devoir s'assurer que tout le monde a la bonne conf sur sa machine :)